persistence/sql/migrations/sql/20220907132836000000_sessions_add_client_fields.cockroach.down.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE sessions
```

Comment:
If many migrations share the same code, you can also create "global" migrations and then override only for databases that have a different DSL:

```patch
- persistence/sql/migrations/sql/20220907132836000000_sessions_add_client_fields.cockroach.down.sql
+ persistence/sql/migrations/sql/20220907132836000000_sessions_add_client_fields.down.sql
```